provisioning/service/samples/create_individual_x509_enrollment

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+'use strict';
+var fs = require('fs');
+var provisioningServiceClient = require('azure-iot-provisioning-service').ProvisioningServiceClient;
+
+var deviceID = process.argv[2];
+var connectionString = process.argv[3];
```

Comment:
since there are now multiple arguments I'd encourage you to look at yargs - and at least add some helper message in case those are undefined.

Reply:
changed it in the new commit